lms/djangoapps/teams/startup.py

```diff
@@ -0,0 +1,22 @@
+"""
+Initialize the mako template lookup
+"""
+from django.conf import settings
+
+from edxmako import LOOKUP
```

Comment:
The mako loaders are working-- at least for our very simple placeholder HTML file! Just have to add "teams" to INSTALLED_APPS, put ## mako at the top of the HTML file, and switch to using django.shortcuts.render_to_response instead of edxmako.shortcuts.render_to_response.